app/src/main/res/layout/item_status.xml

```diff
@@ -13,156 +13,168 @@
+        app:layout_constraintTop_toBottomOf="@id/status_reblogged"
         tools:src="@drawable/avatar_default" />
 
     <com.keylesspalace.tusky.view.RoundedImageView
         android:id="@+id/status_avatar_reblog"
         android:layout_width="24dp"
         android:layout_height="24dp"
-        android:layout_alignBottom="@+id/status_avatar"
-        android:layout_alignEnd="@id/status_avatar"
         android:contentDescription="@null"
         android:visibility="gone"
+        app:layout_constraintBottom_toBottomOf="@id/status_avatar"
+        app:layout_constraintEnd_toEndOf="@id/status_avatar"
         tools:src="@color/accent"
         tools:visibility="visible" />
 
-    <RelativeLayout
-        android:id="@+id/status_name_bar"
-        android:layout_width="match_parent"
+    <androidx.emoji.widget.EmojiTextView
+        android:id="@+id/status_display_name"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_below="@+id/status_reblogged"
-        android:layout_toEndOf="@+id/status_avatar"
-        android:paddingBottom="4dp"
-        android:paddingTop="@dimen/status_avatar_padding">
-
-        <androidx.emoji.widget.EmojiTextView
-            android:id="@+id/status_display_name"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_alignParentStart="true"
-            android:ellipsize="end"
-            android:maxLines="1"
-            android:paddingEnd="@dimen/status_display_name_padding_end"
-            android:paddingStart="0dp"
-            android:textColor="?android:textColorPrimary"
-            android:textSize="?attr/status_text_medium"
-            android:textStyle="normal|bold"
-            tools:text="Ente" />
-
-        <TextView
-            android:id="@+id/status_username"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_toEndOf="@id/status_display_name"
-            android:layout_toStartOf="@+id/status_timestamp_info"
-            android:ellipsize="end"
-            android:maxLines="1"
-            android:textColor="?android:textColorSecondary"
-            android:textSize="?attr/status_text_medium"
-            tools:text="\@Entenhausen" />
+        android:layout_marginStart="14dp"
+        android:ellipsize="end"
+        android:maxLines="1"
+        android:paddingStart="0dp"
+        android:paddingEnd="@dimen/status_display_name_padding_end"
+        android:textColor="?android:textColorPrimary"
+        android:textSize="?attr/status_text_medium"
+        android:textStyle="normal|bold"
+        app:layout_constrainedWidth="true"
+        app:layout_constraintEnd_toStartOf="@id/status_timestamp_info"
+        app:layout_constraintHorizontal_bias="0"
+        app:layout_constraintStart_toEndOf="@id/status_avatar"
+        app:layout_constraintTop_toTopOf="@id/status_avatar"
```

Comment:
to have it look like before, this needs 
```
app:layout_constraintTop_toBottomOf="@id/status_reblogged"
android:layout_marginTop="@dimen/status_avatar_padding"
```
(not sure why this has the name "status_avatar_padding", but thats what was used previously. Feel free to remove and just use 10dp)